x-pack/auditbeat/auditbeat.reference.yml

```diff
@@ -1272,3 +1272,8 @@ logging.files:
 
 # Enable or disable seccomp system call filtering on Linux. Default is enabled.
 #seccomp.enabled: true
+
+#================================= Migration ==================================
```

Comment:
I'm wondering how it's possible that this sneaked in for just the auditbeat file?!? Need to figure out what is wrong in my local setup.